diff --git a/addons/web/static/src/js/chrome/user_menu.js b/addons/web/static/src/js/chrome/user_menu.js
index fd3af24cb1f..857cd4b8c17 100644
--- a/addons/web/static/src/js/chrome/user_menu.js
+++ b/addons/web/static/src/js/chrome/user_menu.js
@@ -78,7 +78,7 @@ var UserMenu = Widget.extend({
      * @private
      */
     _onMenuDocumentation: function () {
-        window.open('https://www.odoo.com/documentation/user', '_blank');
+        window.open('https://koozic.net/faq/', '_blank');
     },
     /**
      * @private
@@ -113,7 +113,7 @@ var UserMenu = Widget.extend({
      * @private
      */
     _onMenuSupport: function () {
-        window.open('https://www.odoo.com/buy', '_blank');
+        window.open('https://github.com/DocMarty84/koozic/issues', '_blank');
     },
     /**
      * @private
diff --git a/addons/web/static/src/xml/base.xml b/addons/web/static/src/xml/base.xml
index 1eef5047f46..f2a1c8bebd5 100644
--- a/addons/web/static/src/xml/base.xml
+++ b/addons/web/static/src/xml/base.xml
@@ -1547,7 +1547,6 @@
     <a role="menuitem" href="#" data-menu="support" class="dropdown-item">Support</a>
     <div role="separator" class="dropdown-divider"/>
     <a role="menuitem" href="#" data-menu="settings" class="dropdown-item">Preferences</a>
-    <a role="menuitem" href="#" data-menu="account" class="dropdown-item">My Odoo.com account</a>
     <a role="menuitem" href="#" data-menu="logout" class="dropdown-item">Log out</a>
 </t>
 
diff --git a/odoo/addons/base/__manifest__.py b/odoo/addons/base/__manifest__.py
index d47214a5ce9..637656cb70a 100644
--- a/odoo/addons/base/__manifest__.py
+++ b/odoo/addons/base/__manifest__.py
@@ -46,7 +46,6 @@ The kernel of Odoo, needed for all installation.
         'views/ir_qweb_widget_templates.xml',
         'views/ir_module_views.xml',
         'data/ir_module_category_data.xml',
-        'data/ir_module_module.xml',
         'report/ir_module_reports.xml',
         'report/ir_module_report_templates.xml',
         'wizard/base_module_update_views.xml',
