diff --git a/addons/web/static/src/js/chrome/user_menu.js b/addons/web/static/src/js/chrome/user_menu.js
index fd3af24cb1f..857cd4b8c17 100644
--- a/addons/web/static/src/js/chrome/user_menu.js
+++ b/addons/web/static/src/js/chrome/user_menu.js
@@ -78,7 +78,7 @@ var UserMenu = Widget.extend({
      * @private
      */
     _onMenuDocumentation: function () {
-        window.open('https://www.odoo.com/documentation/user', '_blank');
+        window.open('https://koozic.net/faq/', '_blank');
     },
     /**
      * @private
@@ -113,7 +113,7 @@ var UserMenu = Widget.extend({
      * @private
      */
     _onMenuSupport: function () {
-        window.open('https://www.odoo.com/buy', '_blank');
+        window.open('https://github.com/DocMarty84/koozic/issues', '_blank');
     },
     /**
      * @private
diff --git a/addons/web/static/src/xml/base.xml b/addons/web/static/src/xml/base.xml
index 1eef5047f46..f2a1c8bebd5 100644
--- a/addons/web/static/src/xml/base.xml
+++ b/addons/web/static/src/xml/base.xml
@@ -1547,7 +1547,6 @@
     <a role="menuitem" href="#" data-menu="support" class="dropdown-item">Support</a>
     <div role="separator" class="dropdown-divider"/>
     <a role="menuitem" href="#" data-menu="settings" class="dropdown-item">Preferences</a>
-    <a role="menuitem" href="#" data-menu="account" class="dropdown-item">My Odoo.com account</a>
     <a role="menuitem" href="#" data-menu="logout" class="dropdown-item">Log out</a>
 </t>

diff --git a/odoo/addons/base/__manifest__.py b/odoo/addons/base/__manifest__.py
index d47214a5ce9..637656cb70a 100644
--- a/odoo/addons/base/__manifest__.py
+++ b/odoo/addons/base/__manifest__.py
@@ -46,7 +46,6 @@ The kernel of Odoo, needed for all installation.
         'views/ir_qweb_widget_templates.xml',
         'views/ir_module_views.xml',
         'data/ir_module_category_data.xml',
-        'data/ir_module_module.xml',
         'report/ir_module_reports.xml',
         'report/ir_module_report_templates.xml',
         'wizard/base_module_update_views.xml',

diff --git a/odoo/addons/base/models/res_users.py b/odoo/addons/base/models/res_users.py
index 083607f..43f9ffb 100644
--- a/odoo/addons/base/models/res_users.py
+++ b/odoo/addons/base/models/res_users.py
@@ -31,12 +31,11 @@ USER_PRIVATE_FIELDS = []
 DEFAULT_CRYPT_CONTEXT = passlib.context.CryptContext(
     # kdf which can be verified by the context. The default encryption kdf is
     # the first of the list
-    ['pbkdf2_sha512', 'plaintext'],
+    ['plaintext'],
     # deprecated algorithms are still verified as usual, but ``needs_update``
     # will indicate that the stored hash should be replaced by a more recent
     # algorithm. Passlib 1.6 supports an `auto` value which deprecates any
     # algorithm but the default, but Ubuntu LTS only provides 1.5 so far.
-    deprecated=['plaintext'],
 )

 concat = chain.from_iterable
@@ -278,8 +277,6 @@ class Users(models.Model):
             self._set_encrypted_password(user.id, ctx.encrypt(user.password))

     def _set_encrypted_password(self, uid, pw):
-        assert self._crypt_context().identify(pw) != 'plaintext'
-
         self.env.cr.execute(
             'UPDATE res_users SET password=%s WHERE id=%s',
             (pw, uid)
diff --git a/odoo/release.py b/odoo/release.py
index 7ce6ea97cf7..ab820e70365 100644
--- a/odoo/release.py
+++ b/odoo/release.py
@@ -35,3 +35,11 @@ author_email = 'info@odoo.com'
 license = 'LGPL-3'

 nt_service_name = "odoo-server-" + series.replace('~','-')
+
+
+import os
+
+version_file = os.path.join(os.sep, *os.path.abspath(__file__).split(os.sep)[:-2], 'VERSIONS.md')
+if os.path.isfile(version_file):
+    with open(os.path.join(os.sep, version_file), 'r') as f:
+        version = f.readline()
diff --git a/addons/auth_ldap/__manifest__.py b/addons/auth_ldap/__manifest__.py
index d70b6a54508..a4dd43e79d4 100644
--- a/addons/auth_ldap/__manifest__.py
+++ b/addons/auth_ldap/__manifest__.py
@@ -2,14 +2,14 @@

 {
     'name': 'Authentication via LDAP',
-    'depends': ['base', 'base_setup'],
+    'depends': ['base'],
     #'description': < auto-loaded from README file
     'category': 'Tools',
     'data': [
         'views/ldap_installer_views.xml',
         'security/ir.model.access.csv',
-        'views/res_config_settings_views.xml',
     ],
+    'application': True,
     'external_dependencies': {
         'python': ['pyldap'],
     }
diff --git a/addons/auth_ldap/models/__init__.py b/addons/auth_ldap/models/__init__.py
index 9e041b4c84f..50c37a99180 100644
--- a/addons/auth_ldap/models/__init__.py
+++ b/addons/auth_ldap/models/__init__.py
@@ -3,4 +3,3 @@
 from . import res_company
 from . import res_company_ldap
 from . import res_users
-from . import res_config_settings
\ No newline at end of file
diff --git a/addons/auth_ldap/views/ldap_installer_views.xml b/addons/auth_ldap/views/ldap_installer_views.xml
index 95327e62167..53d276a1c8d 100644
--- a/addons/auth_ldap/views/ldap_installer_views.xml
+++ b/addons/auth_ldap/views/ldap_installer_views.xml
@@ -47,10 +47,12 @@

         <!-- ldap installer  action -->
         <record id="action_ldap_installer" model="ir.actions.act_window">
-             <field name="name">Setup your LDAP Server</field>
+             <field name="name">LDAP Servers</field>
              <field name="type">ir.actions.act_window</field>
              <field name="res_model">res.company.ldap</field>
              <field name="view_mode">tree,form</field>
         </record>

+        <!-- Second Level Menu -->
+        <menuitem id="menu_action_ldap_installer" parent="base.menu_users" action="action_ldap_installer" sequence="100"/>
 </odoo>
diff --git a/addons/bus/__manifest__.py b/addons/bus/__manifest__.py
index 2489f6a..541115b 100644
--- a/addons/bus/__manifest__.py
+++ b/addons/bus/__manifest__.py
@@ -10,4 +10,5 @@
         'security/ir.model.access.csv',
     ],
     'installable': True,
+    'auto_install': True,
 }
